src/spy-performance/summary/print-summary.ts

```diff
@@ -16,21 +16,25 @@ export const perfStatsReset = function () {
 };
 
 export const getPerfSummary = function () {
-  const longestSelectors = Object.entries(selectorsPerfStat.stats)
-    .map(([key, value]) => ({ ...value, key }))
-    .sort((a, b) => b.duration - a.duration);
+  const a = Object.entries(selectorsPerfStat.stats)
```

Comment:
what are those 4 variables?
removed them for now, let me know if they're needed

Reply:
My bad, was an attempt to figure out what was going wrong with types 🙈